src/components/ProductsPage/ProductsPage.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Sidebar from "../Sidebar/Sidebar.component";
+import Products from "../Products/Products.component";
+import products from "../../mocks/products.json";
+import CatList from "../../mocks/product-categories.json";
+
+function ProductsPage() {
+  const [categories, setCategories] = React.useState([]);
```

Comment:
A lot of times it's better and more readable to call a function or property directly, you can add the useState to the imports and would look better
```
 import React, { useSatate } from 'react';
```
and then when using it
```suggestion
  const [categories, setCategories] = useState([]);
```